docs/Tutorials/OpenAPI.md

```diff
@@ -676,15 +737,16 @@ As JSON, this could look as follows:
 ### oneOf, anyOf and allOf Keywords
 
 OpenAPI 3.x provides several keywords which you can use to combine schemas. You can use these keywords to create a complex schema or validate a value against multiple criteria.
-- oneOf – validates the value against exactly one of the subschemas
-- allOf – validates the value against all the subschemas
-- anyOf – validates the value against any (one or more) of the subschemas
 
-You can use the [`Merge-PodeOAProperty`](../../Functions/OpenApi/Merge-PodeOAProperty)` function to merge multiple properties.
+* oneOf – validates the value against exactly one of the sub-schemas
+
+* allOf – validates the value against all the sub-schemas
 
+* anyOf – validates the value against any (one or more) of the sub-schemas
```

Comment:
Only the first bullet point needed the preceding blank new-line 😛 Lines 742/744 can be removed.

Also, the dashes appear to be MS Word loong dashes, which will need replacing with normal dashes (`-`).